rllib/core/optim/examples/dummy_ppo_optimizer.py

```diff
@@ -0,0 +1,139 @@
+import gym
+import torch
+
+from ray.rllib.core.optim.rl_optimizer import RLOptimizer
+from ray.rllib.core.examples.simple_ppo_rl_module import (
+    SimplePPOModule,
+    get_shared_encoder_config,
+)
+from ray.rllib.core.rl_module.torch.tests.test_torch_rl_module import (
+    to_numpy,
+    to_tensor,
+)
+from ray.rllib.policy.sample_batch import SampleBatch
+from ray.rllib.utils.torch_utils import convert_to_torch_tensor
+from ray.rllib.utils.numpy import convert_to_numpy
+
+
+class DummyPPOTorchOptimizer(RLOptimizer):
+    def __init__(self, module, config):
+        super().__init__(module, config)
+
+    def compute_loss(self, batch, fwd_out):
+        adv = batch["rewards"] - fwd_out["vf"]
+        actor_loss = -(fwd_out["logp"] * adv).mean()
+        critic_loss = (adv**2).mean()
+        loss = actor_loss + critic_loss
+        loss_dict = {
+            "total_loss": loss,
+            "actor_loss": actor_loss,
+            "critic_loss": critic_loss,
+        }
+        return loss_dict
+
+    def construct_optimizers(self):
+        return [
+            torch.optim.SGD(self.module.parameters(), lr=self._config.get("lr", 1e-3))
+        ]
+
+
+class DummyPPOTorchTrainer:
+    def __init__(self, config) -> None:
+        module_config = config["module_config"]
+        optimizer_config = {}
+        self._module = SimplePPOModule(module_config)
+        self._rl_optimizer = DummyPPOTorchOptimizer(self._module, optimizer_config)
+        self._optimizers = self._rl_optimizer.construct_optimizers()
```

Comment:
see my comments below on who should hold the actual list of local optimizer objects.